mkdocs.yml

```
@@ -814,10 +814,16 @@ extra:
       url: '/projects/connect'
   # Global variables
   site_display_name: 'Ibexa Documentation'
+  product_base_name: 'Ibexa'
```

```suggestion
  product_name_base: 'Ibexa'
```
That's how you use it in the doc